lib/licensed/sources/gradle.rb

```diff
@@ -101,27 +108,50 @@ def enumerate_dependencies
       def gradle_executable
         return @gradle_executable if defined?(@gradle_executable)
         @gradle_executable = begin
-          gradlew = File.join(config.pwd, "gradlew")
-          return gradlew if File.executable?(gradlew)
-          "gradle" if Licensed::Shell.tool_available?("gradle")
-        end
+                               gradlew = File.join(config.pwd, "gradlew")
```

Comment:
same here, it doesn't look like anythings actually changed.  can you revert this block?